sentry-rails/lib/sentry/rails/action_cable.rb

```diff
@@ -29,11 +29,11 @@ def capture(connection, transaction_name:, extra_context: nil, &block)
             end
           end
 
-          def start_transaction(env, transaction_name)
+          def start_transaction(env, transaction_name, transaction_source)
```

Comment:
Let's align this with `CaptureExceptions#start_transaction` to just pass a scope?

Reply:
yea good idea, thanks